src/net.cpp

```diff
@@ -2959,11 +2960,21 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
     size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    auto sanitizedType = SanitizeString(msg.m_type);
```

Comment:
Do we need to sanitize this? the message types for all outgoing messages are hardcoded in our binary so should all be safe.

Reply:
Agree, will remove it from the tracepoint and logging.